sphinx/cmd/quickstart.py

```diff
@@ -170,10 +170,7 @@ def _has_custom_template(self, template_name: str) -> bool:
               It will be removed in the future without deprecation period.
         """
         template = path.join(self.templatedir, path.basename(template_name))
-        if self.templatedir and path.exists(template):
-            return True
-        else:
-            return False
+        return bool(self.templatedir) and path.exists(template)
```

Comment:
Is this `bool` needed?

Reply:
Yep, as if templatedir is falsy, the check short circuits and would return the value of templatedir, rather than False.

A

Reply:
Understand. Thanks,